Automattic/sensei
# Actions: add a new workflow to help triaging issues

### Changes proposed in this Pull Request

This workflow uses the Repo Gardening action:
https://github.com/marketplace/actions/repository-gardening

This commit enables 2 tasks of that action:

- Gather support references: Adds a new comment with a list of all support references that may get posted on a GitHub issue, and posts about that issue in Slack, in the channel of your choice (currently #jpop-satellite), if it’s gathered more than 10 tickets. This can be a good opportunity to re-prioritize issues / escalate them when we start seeing them getting more and more attention / reports.
- Reply to customers Reminder: sends a Slack message, in the channel of your choice (currently #jpop-satellite), about High Priority issues with a certain amount of tickets added to them (currently 10), when those issues get closed, to remind you to update customers about the now-fixed issue, or add that issue to your list of future bulk-replies to send when a plugin update is pushed.

Internal reference: p9JkDc-Jj-p2

### Testing instructions

This cannot really be tested before it gets merged, but you can see examples of the workflow in use in other repos, like Calypso or Jetpack. Here is an example issue:
https://github.com/Automattic/jetpack/issues/25011#issuecomment-1207704120

### New/Updated Hooks

None

### Deprecated Code

None

### Screenshot / Video

<img width="969" alt="image" src="https://user-images.githubusercontent.com/426388/184835621-8ffe6b60-bf0c-4408-b679-aad26626d998.png">

-----------

<img width="722" alt="image" src="https://user-images.githubusercontent.com/426388/184835729-cfd6eac4-5092-45f4-9ced-09cd9f62d400.png">

-----------

<img width="704" alt="image" src="https://user-images.githubusercontent.com/426388/184835847-ffe01366-cb5e-421f-bff5-9b369697af02.png">

-----------

**Note**: in the screenshots above, the DRIs getting pinged are from WordPress.com, but for Sensei we'd be pinging Sattelite DRIs (`@heysatellite`).
